<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mat4DataView</class>
 <widget class="QWidget" name="Mat4DataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="matrixType">
     <item>
      <property name="text">
       <string>Transform</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Look At</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Perspective</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Orthogonal</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mat4 Editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="editorLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="invertCheckBox">
     <property name="text">
      <string>Invert Matrix</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveChanges">
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
